src/multizimbutton.cpp

```diff
@@ -36,6 +37,8 @@ void MultiZimButton::update_display()
     const auto view = KiwixApp::instance()->getTabWidget()->currentWebView();
     QListWidgetItem* currentItem = nullptr;
     QIcon currentIcon;
+    const int paddigTopBot = CSS::MultiZimButton::QListWidget::paddingVertical * 2;
```

Comment:
paddig (typos can cause some headache when you try to look up some code by memory and then it turns out that it contained a typo you never noticed that's why I pay attention to them).

Reply:
Done